programs/daobet-cli/main.cpp

```diff
@@ -913,9 +913,9 @@ void ensure_keosd_running(CLI::App* app) {
     // This extra check is necessary when running cleos like this: ./cleos ...
     if (binPath.filename_is_dot())
         binPath.remove_filename();
-    binPath.append(key_store_executable_name); // if cleos and keosd are in the same installation directory
+    binPath.append(key_store_executable_name); // if cleos and daobet-wallet are in the same installation directory
     if (!boost::filesystem::exists(binPath)) {
-        binPath.remove_filename().remove_filename().append("keosd").append(key_store_executable_name);
+        binPath.remove_filename().remove_filename().append("daobet-wallet").append(key_store_executable_name);
```

Comment:
This string literal may divert from the key_store_executable_name (taken from KEY_STORE_EXECUTABLE_NAME cmake variable):

```
CMakeLists.txt
44:set(KEY_STORE_EXECUTABLE_NAME  ${PROJECT}-wallet)
```

Better use key_store_executable_name from config.hpp.